source/guide_ezmlm.rst

```diff
@@ -0,0 +1,159 @@
+.. highlight:: console
+
+.. author:: Michael Behrisch <oss@behrisch.de>
+
+.. tag:: lang-c
+.. tag:: mail
+.. tag:: mailinglist
+
+#########
+ezmlm-idx
+#########
+
+.. tag_list::
+
+ezmlm_ is a set of programs to run mailing lists either from a web interface, via mail or from the command line.
+It can be used as an alternative to Mailman.
+This guide covers only the command line and the mail interface, not the web. It does not use a database but just
+flat files. For using a database and the web see the detailed docs.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Mailing lists
+  * Compiling a C program using Makefiles
+
+License
+=======
+
+ezmlm_ was written by Fred Lindberg, Fred B. Ringel, and Bruce Guenter and released under GPLv2.
+
+
+Installation
+============
+
+Step 1
+------
+
+Download the latest version of ezmlm_ from the website https://untroubled.org/ezmlm/, extract it and enter the directory:
+
+::
+
+ [isabell@stardust ~] wget https://untroubled.org/ezmlm/archive/7.2.2/ezmlm-idx-7.2.2.tar.gz
+ [isabell@stardust ~] tar -xzf ezmlm-idx-7.2.2.tar.gz
+ [isabell@stardust ~] cd ezmlm-idx-7.2.2/
+ [isabell@stardust ezmlm-idx-7.2.2]
+
+Step 2
+------
+
+Change the configuration by editing the ``conf-*`` files using a text editor. It is usually sufficient to change the following files: ``conf-bin`` to
+
+::
+
+ /home/isabell/bin
+
+``conf-etc`` to
+
+::
+
+ /home/isabell/etc/ezmlm
+
+``conf-lib`` to
+
+::
+
+ /home/isabell/lib/ezmlm
+
+``conf-man`` to
+
+::
+
+ /home/isabell/man
+
+Then run
+
+::
+
+ [isabell@stardust ezmlm-idx-7.2.2] make
+ [...]
+ [isabell@stardust ezmlm-idx-7.2.2] make man
+ [...]
+ [isabell@stardust ezmlm-idx-7.2.2] ./ezmlm-test
+ getconfopt library:   OK
+ ezmlm-make:           OK
+ Using subdb plugin:   std
+ [...]
+ ezmlm-send:           OK
+ ezmlm-send (from):    OK
+ ezmlm-send trailer:   OK
+ omitbottom (-manage): OK
+ omitbottom (-get):    OK
+ Cleaning up...
+ [isabell@stardust ezmlm-idx-7.2.2]
+
+
+The test call should show that everything is OK. Also ``man ezmlm-make`` should work.
```

Comment:
This fails for me, because `~/man` is empty (so is `~/bin`). I guess this is missing a `make install`, which fails due to missing directories:

```
./installer "`head -n 1 conf-bin`" < BIN
./installer "`head -n 1 conf-etc`" < ETC
./installer "`head -n 1 conf-man`" < MAN
./installer "`head -n 1 conf-lib`" < LIB
installer: fatal: unable to mkdir /home/isabell/lib/ezmlm: file does not exist
make: *** [install] Error 111
```

After a quick `mkdir ~/lib` + `make install`, it works just fine. Can you please add those missing steps? :)